upmc-enterprises/cda2fhir
# [UPMCFHIR-188] Formatting.

#### What does this PR do?
Inflicts formatting across all files.

#### Related JIRA tickets:
UPMCFHIR-188

#### How should this be manually tested?
Aside from running the Junit tests on the off chance that something broke, another safe check is to try and merge it to an existing PR to see how difficult it is to merge with that. My theory is that as we've applied the save action to our files, the merging should be smoother. 

#### Background/Context:

#### New JIRA tickets for clinical review required?
